pelican-theme/setup.py

```diff
@@ -0,0 +1,42 @@
+
+from setuptools import setup, find_packages
+
+
+setup(
+    name="mcss-theme",
+    version=0.0.1,
+    description="A Pelican theme using m.css",
+    long_descri
+    author="W. Minchin",
+    author_email="w_minchin@hotmail.com",
+    url="https://github.com/MinchinWeb/seafoam",
+    packages=find_packages(),
+    include_package_data=True,
+    install_requires=[
+        'pelican',
+        'minchin.pelican.plugins.image_process>=1.0.1, !=1.1.2',
+        'minchin.pelican.jinja_filters',
+        # requires asset plugin, bundle? -- https://github.com/getpelican/pelican-plugins/tree/master/assets
+        ],
+    extras_require={
+        ':python_version < "3.4"': ['pathlib2'],
+        'dev': ['minchin.releaser',
```

Comment:
The `minchin` package is your personal stuff, I suppose?

Reply:
lol, no that's mine :)